### ISSUE TITLE
Teste Fake

### ISSUE THREAD

**Issue by miquelven** [MKS-desenvolvimento-de-sistemas/mks-frontend-challenge#42]
Teste fake, não entreguem nada galera.

Mais informações: https://www.linkedin.com/feed/update/urn:li:activity:7176954115560792065/

**Comment by Kleber-JF** [comment]
Olá, o link que vc enviou me redireciona pra um post nada a ver com esse teste, poderia me dar mais detalhes sobre?
Eu estava aqui fazendo o teste e a pessoa que me enviou me avisou sobre possiveis comentarios tentando falar que o teste era falso.
E não consigo achar nada falando sobre.
Obrigado!

**Comment by miquelven** [comment]
Oi, devo ter colocado o link errado, o certo seria esse: https://www.linkedin.com/posts/vinianhaiadev_normalmente-eu-uso-o-linkedin-para-postagens-activity-7130297930044874755-hzXV/

**Comment by Kleber-JF** [comment]
Muito obrigado!
Pude perceber que algumas pessoas receberam feedback do teste, mas eu acabei suspeitando de talvez ser uma empresa ilegal querendo contratar gente honesta pra negócios obscuros.
Vlw!

**Comment by juanevangelista1** [comment]
Pessoal, boa noite. Vi a publicação e já estava com um pé atrás porque foi a primeira vez que um recrutador manda mensagem e avisa sobre tentativas de candidatos descredibilizarem o processo e a empresa. Realmente... Achei estranho

**Comment by Kleber-JF** [comment]
Exatamente, ele falou a mesma coisa pra mim, tb achei estranho e acho que faltam informações da empresa.
